javascript/javascript1/week1/script.js

```diff
@@ -25,4 +25,43 @@
         console.log (`Your dog will be ${dogYear} human years old in ${dogYearFuture}`);
     }
 
+// Housey pricey (A house price estimator)
+    let detailsOfHouse = [
+        {
+            name: "Peter", 
+            length: 10,
+            breadth: 8,
+            height: 10,  
+            gardenSizeInM2: 100,    
+            houseCost: 2500000,
+        },
+        {
+            name: "Julia", 
+            length: 11,
+            breadth: 5,
+            height: 8,  
+            gardenSizeInM2: 70,    
+            houseCost: 1000000,
+        },
+    ]
+    
+    // Using index number to access details of "Peter" or "Julia"
+    let indexNumber = 0;
+    
+    // Calculation volumeInMeters = length * breadth * height;
+    const volumeInMeters = (detailsOfHouse[indexNumber].length) * (detailsOfHouse[indexNumber].breadth) * (detailsOfHouse[indexNumber].height);
+    
+    // Calculation const housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+    const housePrice = volumeInMeters * 2.5 * 1000 + (detailsOfHouse[indexNumber].gardenSizeInM2) * 300;
+    console.log(housePrice);
+    
+    //Condition based on comparision between housePrice and houseCost
```

Comment:
This is very good, and clear to understand, you are just missing Julia . We need to find out how much she is going to pay. I know it is just repeat of what you have already done , but it is part of the homework.

Reply:
Actually, I didn't miss Julie's details. I have created the variable indexNumber. When I have submitted the code its value is 0 and the details of Peter's house is visible in console. But if its changed to 1, the details of Julie's house is visible in console.

This was actually the whole purpose of using variable indexNumber. 
Is it ok to do it this way or should I write separate code to view Julie's details.